src/languages/erlang.js

```diff
@@ -80,9 +80,39 @@ export default function(hljs) {
     scope: 'string',
     match: /\$(\\([^0-9]|[0-9]{1,3}|)|.)/,
   };
+  const TRIPLE_QUOTE = hljs.END_SAME_AS_BEGIN({
```

Comment:
Since it looks like this has no sub-rules, can this be done with pure regex instead?  JS matches are very slow comparatively speaking.

Reply:
Not too familiar with that, I tried `/(?<open>"{3,}).*(\k<open>)/` but it didn't seem to work as I hoped.
Do you know another language syntax I can look at for reference?

Reply:
I _think) just the usual `\1` to reference a prior match group should work, you don't need any fancy named groups, etc...  Pretty sure if you dig around in the other grammars for strings you'll see this used in a few places.

Reply:
You're right, I found some previous work in csharp.js and used that.